contracts/exchange/contracts/src/MixinMatchOrders.sol

```diff
@@ -267,6 +268,48 @@ contract MixinMatchOrders is
         return matchedFillResults;
     }
 
+    function _dispatchTransferFrom(
```

Comment:
How are these declarations used?

Reply:
This (as well as all other abstract functions that I added) are needed to specify the functions in the Mixin. In this case, `_dispatchTransferFrom` and `_getCurrentContextAddress` are used in the private function at the end of the file. 

We could eliminate some of these functions by creating special purpose interfaces for them. I didn't do this initially because it felt like doing this was sort of against the spirit of these changes. This said, I'm happy to try and come up with some interfaces that we could use like that.

Reply:
Ohh, I see what's going on. So declarations in `MFoobar` would need to be replaced by their respective implementation, defined in `MixinFoobar`. In this case, `MixinMatchOrders` would now inherit from `MixinAssetProxyDispatcher` rather than `MAssetProxyDispatcher`.